lib/utils/file.js

```diff
@@ -6,12 +6,14 @@
 // Issue: https://github.com/kevva/executable/issues/9
 // PR: https://github.com/kevva/executable/pull/10
 
-export function isExecutable (fileStat) {
-  if (process.platform === 'win32') return true;
+export function isExecutable(fileStat) {
+  if (process.platform === 'win32') {
+    return true;
+  }
 
   return Boolean(
-    (fileStat['mode'] & parseInt('0001', 8)) ||
-    (fileStat['mode'] & parseInt('0010', 8)) ||
-    (fileStat['mode'] & parseInt('0100', 8))
+    (fileStat.mode & parseInt('0001', 8)) ||
+    (fileStat.mode & parseInt('0010', 8)) ||
+    (fileStat.mode & parseInt('0100', 8))
```

Comment:
No rule for this yet, but these should use octal literals instead of `parseInt`.

http://www.2ality.com/2015/04/numbers-math-es6.html